content/blog/2020/12/2020-12-05-gsod-project-report.adoc

```diff
@@ -0,0 +1,117 @@
+* Contacting Stakeholders and onboarding contributors: The project was  announced on social media and different Jenkins channels. 
+I wrote a link:https://www.jenkins.io/blog/2020/09/25/document-jenkins-on-kubernetes-introduction/[blog post] to announce the project and created a link:https://www.jenkins.io/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes/[project page] on Jenkins.io.
+* Knowledge transfer: I and my mentors planned knowledge sharing sessions and fixed tentative dates based on the availability of the trainers. 
+My mentors also shared useful resources to help me prepare for the project.
+* Getting permissions: I and my mentors agreed I didn’t need any special permissions from the beginning, however, this topic was left open for discussion if the need arose later on in the project.
+* Pre-planning of the project: I refined my goals and set expectations with my mentor and also learned more about the community(Jenkins). 
+I also had to ensure that the proposed documentation structure I drafted was in line with the goals of the organization so my mentors vetted it and we finalized on the proposed sections that I was supposed to work on. 
+
+
+## Documentation Development Phase
+
+
+### Knowledge Sharing Sessions
+
+During the development phase, my mentors hosted two knowledge sharing sessions:
+
+* link:https://youtu.be/BkIiGXDCEGA[Katakoda and Helm] by link:https://twitter.com/markyjackson5[Marky Jackson], you can find the slides for this session link:https://docs.google.com/presentation/d/1LD7btYQaSiI3R8226OIAI6EIUqRZHTyjefVMdF4ydSg/edit?usp=sharing[here]. 
+* link:https://youtu.be/9WIGVLBIfNM[Helm] by link:https://twitter.com/torsten_walter[Torsten Walter], you can find the slides for this session link:https://docs.google.com/presentation/d/1RS8PwlR_FzxYypBlwtp4LcZls8hr3dG_4KJ65U00Xlo/edit#slide=id.gc6f80d1ff_0_0[here]. 
+
+These sessions gave me an in-depth understanding of concepts and tools needed for the project.
+
+
+### Jenkins on Kubernetes Documentation Skeleton
+
+At the application phase, I drafted a link:https://docs.google.com/document/d/1wMeeN4oA7AN4F3pfLBIAJZWXD7PdqSKHotdk76yCw68/edit?usp=sharing[structure] describing the proposed Jenkins on Kubernetes section. 
+My intention was to use it as a guide during the implementation phase of the project, but when the development phase kicked off, my mentors and I thought of a better approach to creating a new Jenkins on Kubernetes section which was to add the Jenkins on Kubernetes contents to existing related sections for easier navigation and better user experience. 
+An example of this approach would be creating the link:https://www.jenkins.io/doc/book/installing/kubernetes/[Installing Jenkins on Kubernetes] section under the link:https://www.jenkins.io/doc/book/installing/[Installing Jenkins] section rather than putting it under an entirely new section.  
```

Comment:
```suggestion
An example of this approach would be creating the link:/doc/book/installing/kubernetes/[Installing Jenkins on Kubernetes] section under the link:/doc/book/installing/[Installing Jenkins] section rather than putting it under an entirely new section.  
```